scripts/core/usm.yml

```diff
@@ -70,6 +70,8 @@ etors:
       desc: "Memory allocation size info"
     - name: DEVICE
       desc: "Memory allocation device info"
+    - name: POOL_HANDLE
```

Comment:
The `DEVICE` is also a handle, `POOL` without `_HANDLE` would be consistent with the other property names.

Reply:
Good point, I changed it back.